javascript/ecmascript-2015/generators-proxy/proxy-interception-points.md

```diff
@@ -35,7 +35,7 @@ Proxies allow you to intercept many different operations and methods:
 * ownKeys
 * apply
 
-Of course you only need to specify those you want to intercept otherwise the default behavior will occur.
+Of course you only need to specify those you want to intercept otherwise the default behaviour will occur.
```

Comment:
Hi @Therrance, this isn't really a needed change because it comes down to the [difference between British vs American English](https://www.grammar.com/behavior_vs._behaviour) and since a majority of programming languages and literature follow the American English conventions, that's what we're doing as well.